kstream/kstream-interactive-query/README.adoc

```diff
@@ -0,0 +1,46 @@
+== What is this app?
+
+This is an example of a Spring Cloud Stream processor using Kafka Streams support.
+
+The example is based on a contrived use case of tracking products by interactively querying their status.
+The program accepts product ID's and track their counts hitherto by interactively querying the underlying store. \
+This sample uses lambda expressions and thus requires Java 8+.
+
+==== Starting Kafka in a docker container
+
+* Skip steps 1-3 if you already have a non-Docker Kafka environment.
+
+1. Go to the docker directory in this repo and invoke the command `docker-compose up -d`.
+2. Ensure that in the docker directory and then invoke the script `start-kafka-shell.sh`
```

Comment:
This could be a very basic question. I am trying to understand how is this `start-kafka-shell.sh` different from the `docker-compose up`? The `up` command would have already had the `docker` containers running and not sure what does this `start-kafka-shell.sh` do after that.

Reply:
@ilayaperumalg `docker-compose up` basically spins up the kafka cluster. `start-kafka-shell.sh` gives you access to the shell environment so that you can invoke the producer/consumer scripts.